src/sgame/sg_active.cpp

```diff
@@ -2196,28 +2195,9 @@ void ClientThink_real( gentity_t *self )
 
 			ent->use( ent, self, self ); // other and activator are the same in this context
 		}
-		else
+		else if ( client->pers.team == TEAM_ALIENS )
 		{
-			// no entity in front of player - do a small area search
-			for ( ent = nullptr; ( ent = G_IterateEntitiesWithinRadius( ent, client->ps.origin, ENTITY_USE_RANGE ) ); )
-			{
-				if ( ent && ent->use && ent->buildableTeam == client->pers.team)
-				{
-					if ( g_debugEntities.Get() > 1 )
-					{
-						Log::Debug("Calling entity->use after an area-search for %s", etos(ent));
-					}
-
-					ent->use( ent, self, self ); // other and activator are the same in this context
-
-					break;
-				}
-			}
```

Comment:
I believe deleting this brings us closer to have interactive usable map entities.

Reply:
wait, further, I meant that deleting this brings us further from usable map entities